src/RollImage.cpp

```diff
@@ -4383,21 +4383,29 @@ void RollImage::setMidiFileTempo(MidiFile& midifile) {
 		// TPQ is 6 times the tempo at 300 DPI, (so 591 = 6 * 98.5)
 		// 591 is for Red Welte tempo of 98.5 (~3 meters/minute)
 		// 568 is the currently preferred value, however.
+		// Wayne Stahnke uses 47.2 mm/s (1.858 in/s) = 557.4 TPQ
 		midifile.setTPQ(568);
 		setRollAcceleration(.30865);
 	} else if (m_rollType == "welte-green") {
 		// Peter Phillips's dissertation describes the speed for green Welte
-		// rolls as "seven feet per minute" (p. 121). 7ft/min = 2.1336m/min =
+		// rolls as "seven feet per minute" (p. 121). 7 ft/min = 2.1336 m/min =
 		// tempo of 70.0532 (* 6 = 420).
 		midifile.setTPQ(420);
-		setRollAcceleration(.22336);
+		setRollAcceleration(.16896);
+	} else if (m_rollType == "welte-licensee") {
+		// This is equivalent to the median speed of "eight feet per minute"
+		// that Peter Phillips mentions for the De Luxe Welte Licensee rolls 
+		// (p. 126): 8 ft/min = 2.438 m/min = tempo of 80.0608.
+		// Wayne Stahnke suggests a speed of 1.7 in/s = 8.5 ft/min = 510 TPQ
+		midifile.setTPQ(6 * 80);
+		// XXX Acceleration has not been modeled/observed. Wayne Stahnke
+		// projects that the effective spool diameter for Licensee is ~90.9%
+		// of the Green spool, so assuming this translates to acceleration:
+		setRollAcceleration(.16896 * 0.909);
```

Comment:
I don't know if can/should do anything more detailed with this, and at some point we have to pick a number and run with it, but shouldn't a smaller effective diameter for the spool result in a greater acceleration factor (with each revolution of the pick-up spool the effective diameter is increased by a greater percentage of it's original value)?  This becomes less important if we can expose this in the Pianolatron app, but perhaps it bears a little more thought in this context at some point in the future.  (Happy to merge this in the mean time.)

Reply:
Good catch -- if the takeup spool's circumference were equal to the length of the roll, we wouldn't have to worry about acceleration at all! So presumably the `*` should be a `/` here.
Better default numbers are on the way, assuming I do some math that I've been putting off for a while, but after we've settled on a reasonable starting point, being able to expose this factor for tuning in the Pianolatron app would be the ideal situation.

Reply:
Yeah, I couldn't immediately wrap my head around whether it would appropriate simply to divide by .909 instead, tbh (should it be `1 / (π * .909)` ? 😕), so I just flagged it and merged...  It would be good to have a "best guess" here (and for the default in the Pianolatron context, too).